client/src/main/java/com/chattycathy/client/handler/ClientStompSessionHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.chattycathy.client.handler;
+
+import com.chattycathy.client.model.Model;
+import io.micrometer.common.lang.NonNullApi;
+import io.micrometer.common.lang.Nullable;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.messaging.simp.stomp.StompCommand;
+import org.springframework.messaging.simp.stomp.StompHeaders;
+import org.springframework.messaging.simp.stomp.StompSession;
+import org.springframework.messaging.simp.stomp.StompSessionHandlerAdapter;
+import org.springframework.stereotype.Component;
+
+import java.lang.reflect.Type;
+
+@NonNullApi
+@Component
+@Slf4j
+public class ClientStompSessionHandler extends StompSessionHandlerAdapter {
+
+    @Override
+    public void afterConnected(StompSession session, StompHeaders connectedHeaders) {
+        log.info("Connected to WebSocket server");
+        log.debug("Connected successfully to session {}, headers: {}", session, connectedHeaders);
+
+        session.subscribe("/topic/ping", this);
+
+        Model model = new Model("Client", "Hi Chatty Cathy Server");
+
+        session.send("/app/ping", model);
+    }
+
+    @Override
+    public void handleException(StompSession session, @Nullable StompCommand command, StompHeaders headers, byte[] payload, Throwable exception) {
+        log.error("Error: {}", exception.getMessage());
```

Comment:
I believe using `exception.getMessage()` will remove the stack trace from being logged which will make debugging harder.

```suggestion
        log.error("Error: ", exception);
```

Reply:
This was an intentional decision. The errors will be from packages like the JacksonConverter stuff so the message itself is all the description we need.

Say for instance model is: `name: string, id: integer` and we got `name: string, id: long`, the error message would tell us that there is this mismatch and it's on the `id` attribute. The stacktrace in this case will not be useful.

Is this acceptable to just keep the error message for? @bellaisgroovy